src/index.js

```diff
@@ -41,6 +41,7 @@ class LineClamp extends Component {
         background = '#fff',
         children,
         ellipsis = '...',
+        showEllipsis = true,
```

Comment:
:thinking: If we did this as the inverse (`noEllipsis`) then we could use this as a flag instead and wouldn't have to set a default value.
###### As is:
`<LineClamp showEllipsis={false} />`
###### As a flag:
`<LineClamp noEllipsis />`